File: kotlin-ide/j2k/old/testData/fileOrElement/methodCallExpression/specialBuiltinMembers.java
import java.util.*;

enum E {
    A, B, C
}

class A {
    void foo(List<String> list, Collection<Integer> collection, Map<Integer, Integer> map) {
        int a = "".length();
        String b = E.A.name();
        int c = E.A.ordinal();
        int d = list.size() + collection.size();
        int e = map.size();
        int f = map.keySet().size();
        int g = map.values().size();
        int h = map.entrySet().size();
    }

    void bar(List<String> list, HashMap<String, Integer> map) {
        char c = "a".charAt(0);
        byte b = new Integer(10).byteValue();
        int i = new Double(10.1).intValue();
        float f = new Double(10.1).floatValue();
        long l = new Double(10.1).longValue();
        short s = new Double(10.1).shortValue();

        try {
            String removed = list.remove(10);
            Boolean isRemoved = list.remove("a");
        }
        catch(Exception e) {
            System.err.println(e.getMessage());
            throw new RuntimeException(e.getCause());
        }

        for (Map.Entry<String, Integer> entry : map.entrySet()) {
            String key = entry.getKey();
            Integer value = entry.getValue();
            entry.setValue(value + 1);
        }
    }
}

File: kotlin-ide/j2k/old/testData/fileOrElement/methodCallExpression/specialBuiltinMembers.kt
import java.util.*

internal enum class E {
    A, B, C
}

internal class A {
    fun foo(list: List<String>, collection: Collection<Int>, map: Map<Int, Int>) {
        val a = "".length
        val b = E.A.name
        val c = E.A.ordinal
        val d = list.size + collection.size
        val e = map.size
        val f = map.keys.size
        val g = map.values.size
        val h = map.entries.size
    }

    fun bar(list: MutableList<String>, map: HashMap<String, Int>) {
        val c = "a"[0]
        val b = 10.toByte()
        val i = 10.1.toInt()
        val f = 10.1.toFloat()
        val l = 10.1.toLong()
        val s = 10.1.toShort()

        try {
            val removed = list.removeAt(10)
            val isRemoved = list.remove("a")
        } catch (e: Exception) {
            System.err.println(e.message)
            throw RuntimeException(e.cause)
        }

        for (entry in map.entries) {
            val key = entry.key
            val value = entry.value
            entry.setValue(value + 1)
        }
    }
}
